SampleUsages/TestSamples/Http-CSharp-CPUIntensive/azuredeploy.json

```diff
@@ -0,0 +1,105 @@
+﻿{
```

Comment:
azuredeploy.json is same as node sample, except for name change in functionAppName & hostingPlanName